### PR TITLE
Migrate `sci` plugin to `builtins`

### PR DESCRIPTION
**Pull request by jmcook1186**
### Types of changes

- [x] Enhancement (project structure, spelling, grammar, formatting)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### A description of the changes proposed in the Pull Request

Migrates SCI plugin code into `if/builtins`
Addresses #682

### CODE REVIEW

File: src/builtins/sci/README.md [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+name: sci-demo
+description: example invoking sci plugin
+tags:
+initialize:
+  outputs:
+    - yaml
+  plugins:
+    sci:
+      method: Sci
+      path: '@grnsft/if-plugins'
```
Comment (manushak) [inline src/builtins/sci/README.md:140]: can you change the path here too?

File: src/builtins/sci/README.md [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+sci-per-s = carbon-operational + carbon-embodied / duration  // (= 5.02)
+sci-per-minute = sci-per-s * 60  // (= 301.2)
+sci-per-functional-unit-time = sci-per-minute * number of minutes
+sci-per-f-unit = sci-per-functional-unit-time / 100  // (= 3.012 gC/request)
+```
+
+To run the plugin, you must first create an instance of `Sci`. Then, you can call `execute()` to return `sci`.
+
+```typescript
+import {Sci} from '@grnsft/if-plugins';
```
Comment (manushak) [inline src/builtins/sci/README.md:105]: change import

File: src/builtins/sci/README.md [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+          carbon-embodied: 5
+          duration: 1
+          requests: 100
+```
+
+You can run this example `manifest` by saving it as `./examples/manifests/test/sci.yml` and executing the following command from the project root:
+
+```sh
+npm i -g @grnsft/if
+npm i -g @grnsft/if-plugins
```
Comment (manushak) [inline src/builtins/sci/README.md:163]: remove this line

File: src/builtins/sci/README.md [hunk 4]
```diff
@@ -0,0 +1,167 @@
...
+          functional-unit: requests # factor to convert per time to per f.unit
+      inputs:
+        - timestamp: 2023-07-06T00:00
+          carbon-operational: 0.02
+          carbon-embodied: 5
+          duration: 1
+          requests: 100
+```
+
+You can run this example `manifest` by saving it as `./examples/manifests/test/sci.yml` and executing the following command from the project root:
```
Comment (manushak) [inline src/builtins/sci/README.md:159]: change manifest path

File: src/builtins/sci/README.md (hunk 1 above)
Comment (jmcook1186) [reply src/builtins/sci/README.md:140]:
```suggestion
      path: '@grnsft/builtin'
```

File: src/builtins/sci/README.md (hunk 2 above)
Comment (jmcook1186) [reply src/builtins/sci/README.md:105]:
```suggestion
import {Sci} from '@grnsft/if-builtin';
```

File: src/builtins/sci/README.md (hunk 4 above)
Comment (jmcook1186) [reply src/builtins/sci/README.md:159]:
```suggestion
You can run this example `manifest` by saving it as `./manifests/plugins/sci.yml` and executing the following command from the project root:
```

File: src/builtins/sci/README.md [hunk 5]
```diff
@@ -0,0 +1,166 @@
...
+name: sci-demo
+description: example invoking sci plugin
+tags:
+initialize:
+  outputs:
+    - yaml
+  plugins:
+    sci:
+      method: Sci
+      path: '@grnsft/builtin'
```
Comment (jmcook1186) [inline src/builtins/sci/README.md:140]:
```suggestion
      path: 'builtin'
```

File: src/builtins/sci/README.md [hunk 6]
```diff
@@ -0,0 +1,166 @@
...
+sci-per-s = carbon-operational + carbon-embodied / duration  // (= 5.02)
+sci-per-minute = sci-per-s * 60  // (= 301.2)
+sci-per-functional-unit-time = sci-per-minute * number of minutes
+sci-per-f-unit = sci-per-functional-unit-time / 100  // (= 3.012 gC/request)
+```
+
+To run the plugin, you must first create an instance of `Sci`. Then, you can call `execute()` to return `sci`.
+
+```typescript
+import {Sci} from '@grnsft/if-builtin';
```
Comment (jmcook1186) [inline src/builtins/sci/README.md:105]:
```suggestion
import {Sci} from 'builtins';
```

**Review by narekhovhannisyan — APPROVED** [review]

**Review by manushak — APPROVED** [review]

File: src/builtins/sci/README.md [hunk 7]
```diff
@@ -0,0 +1,166 @@
...
+name: sci-demo
+description: example invoking sci plugin
+tags:
+initialize:
+  outputs:
+    - yaml
+  plugins:
+    sci:
+      method: Sci
+      path: 'builtins'
```
Comment (manushak) [inline src/builtins/sci/README.md:140]: should be `builtin`

Comment (jmcook1186) [reply src/builtins/sci/README.md:140]:
```suggestion
      path: 'builtin'
```

**Pull request merged**